### PR TITLE
Update sort.md

### PR DESCRIPTION
**Pull request by Larry850806**

### CODE REVIEW

File: pages/common/sort.md [hunk 1]
```diff
@@ -13,3 +13,11 @@
 - Sort passwd file by the 3rd field:
 
 `sort -t: -k 3n /etc/passwd`
+
+- Sort a file as number in ascending order:
+
+`sort -n {{filename}}`
+
+- Sort a file and delete duplicate lines:
```
Comment (waldyrious) [inline pages/common/sort.md]: It would be nice to use the word "unique" here to provide a mnemonic for the option name -u.

Comment (Larry850806) [inline pages/common/sort.md]: Can you help me to modify that?

Comment (waldyrious) [inline pages/common/sort.md]: Do you mean a suggestion for a different wording, or as in how to perform such a change using git?

Comment (Larry850806) [inline pages/common/sort.md]: Yes, a suggestion for a different wording.

Comment (waldyrious) [inline pages/common/sort.md]: Perhaps "Sort a file preserving only unique lines"?

**Comment by waldyrious** [comment]
Discussion is continued in #841.

**Pull request closed without merging**